src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -650,6 +659,36 @@ internal PropertyMirror(ProcedureNode procNode, bool isSetter = false)
                 this.Name = PropertyName;
                 this.isSetter = isSetter;
             }
+
+            private string Argument
+            {
+                get
+                {
+                    var args = procNode.ArgumentInfos.Any()
+                        ? " (" + string.Join(", ",
+                                    thisArgName + " : " + procNode.ArgumentTypes[0].ToString().Split('.').Last())
+                               + ')' : "";
+                    return args;
+                }
+            }
+
+            /// <summary>
+            /// For an instance property of a class, called "Property"
+            /// this returns the string
+            /// "Property : 'return type' (this : 'class type')."
+            /// For a static property of a class,  this returns
+            /// "Property : 'return type'."
+            /// </summary>
+            /// <returns></returns>
+            public override string ToString()
+            {
+                var sb = new StringBuilder();
+
+                var returnType = " : " + procNode.ReturnType.ToString().Split('.').Last();
```

Comment:
is this standard, that we only return the typeName not the fully qualified type name? Or am I misinterpreting this?

Reply:
We display just the type in the function signature insight window just to be less verbose.

Reply:
it just seems this could be confusing when dealing with multiple `Point` types for example - I guess we could use the elementResolver or VM to determine the shortest qualified name that is explicit? Maybe a future improvment?

Reply:
Yes, we could. For now, I've made it consistent with method overload completion in the insight window:
![image](https://user-images.githubusercontent.com/5710686/101409804-68e06980-38ac-11eb-864e-5ca0cb42b7e9.png)